src/CurrentAndNextFeeSchedule.js

```diff
@@ -0,0 +1,62 @@
+import * as proto from "@hashgraph/proto";
+export default class CurrentAndNextFeeSchedule {
+    /**
+    * @param {object} props
+    * @param {object} props.currentFeeSchedule
+    * @param {object} props.nextFeeSchedule
+    */
+   
+    constructor(props) {
+        /*
+        * Contains current Fee Schedule
+        *
+        * @type {object}
+        */
+        this.currentFeeSchedule = props.currentFeeSchedule;
+
+        /*
+        * Contains next Fee Schedule
+        *
+        * @type {object}
+        */
+        this.nextFeeSchedule = props.nextFeeSchedule;
+    }
+
+   /**
+     * @param {Uint8Array} bytes
+     * @returns {CurrentAndNextFeeSchedule}
+    */
+    static fromBytes(bytes) {
+        return CurrentAndNextFeeSchedule.fromProtobuf(proto.CurrentAndNextFeeSchedule.decode(bytes));
+    }
+
+    /**
+    * @param {proto.ICurrentAndNextFeeSchedule} currentAndNextFeeSchedule
+    * @returns {CurrentAndNextFeeSchedule}
+    */
+    static fromProtobuf(currentAndNextFeeSchedule) {
+        return new CurrentAndNextFeeSchedule({
+            /** @type {object} */
+            currentFeeSchedule: (currentAndNextFeeSchedule.currentFeeSchedule),
+            /** @type {object} */
+            nextFeeSchedule: (currentAndNextFeeSchedule.nextFeeSchedule),
+        });
+    }
+
+   /**
+     * @returns {proto.ICurrentAndNextFeeSchedule}
+     */
+    toProtobuf() {
```

Comment:
Should be private with `_`. Not sure if I mentioned this earlier, but "private" method in JS are simply start with `underscore` and have an `@internal` in the JS Doc